Agents/VisBackendAgent/agent/src/main/java/com/cmclinnovations/agent/template/FormTemplateFactory.java

```diff
@@ -104,27 +100,7 @@ private void sortData(ArrayNode fields) {
       if (field.has(ShaclResource.TYPE_KEY)) {
         String type = field.path(ShaclResource.TYPE_KEY).get(0).asText();
         if (type.equals(ShaclResource.SHACL_PREFIX + ShaclResource.PROPERTY_SHAPE)) {
```

Comment:
```suggestion
        if (type.equals(ShaclResource.SHACL_PREFIX + ShaclResource.PROPERTY_SHAPE)) {
```was this meant to be deleted?